sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_AnalyzeOperationAsync_AutomaticPolling.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.AI.TextAnalytics.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    [LiveOnly]
+    public partial class TextAnalyticsSamples: SamplesBase<TextAnalyticsTestEnvironment>
+    {
+        [Test]
+        public async Task AnalyzeOperationAsync_AutomaticPolling()
```

Comment:
What about.. 
```suggestion
        public async Task AnalyzeOperationAsync_AutomaticPollingWithSetInterval()
```
or something that suggest this is not the ultimate automated way

Reply:
same comment for the healthcare method